cyfhir/src/main/java/com/Optum/CyFHIR/models/FhirRecursiveObj.java

```diff
@@ -0,0 +1,27 @@
+package com.Optum.CyFHIR.models;
+
+
+import org.neo4j.graphdb.Node;
+
+import java.util.ArrayList;
+
+public class FhirRecursiveObj {
+    Node node;
+    ArrayList<FhirRelationship> relationships;
+
+    public Node getNode() {
+        return node;
+    }
+
+    public ArrayList<FhirRelationship> getRelationships() {
+        return relationships;
+    }
+
+    public void setNode(Node node) {
```

Comment:
If we're sticking with Java, we should consider Lombok for a future task

Reply:
That is a good idea!

Reply:
Agreed, Lombok would reduce a lot of boilerplate in Java.